src/HPWHpresets.cc

```diff
@@ -515,7 +518,7 @@ int HPWH::HPWHinit_presets(MODELS presetNum) {
 
 	else if (presetNum == MODELS_StorageTank) {
 		setNumNodes(12);
-		setpoint_C = 800.;
+		setpoint_C = F_TO_C(127.);
```

Comment:
I think I was confused before. I don't think the setpoint should change (assuming 800 was chosen to do something to disable the heat source controls). Instead, the initial temperature for this preset should be set to 127F.